src/components/PaginationPost.module.scss

```diff
@@ -0,0 +1,80 @@
+@use '../styles/_variables.scss';
```

Comment:
@kevcenteno I created this file as well as `PaginationPost.module.scss`. Both files have a lot of duplicate code so I think we might be able to reduce this, but I'm having trouble figuring out the best way to do this.

Reply:
You can import `Pagination.module.scss` file in both pagination components and create+use a parent `.paginationPost` class that modifies the default styling for the `paginationPost` component.

Reply:
Thank you! This is ready for another review.